CRM/Event/Form/Participant.php

```diff
@@ -1436,8 +1436,8 @@ public function buildEventFeeForm($form) {
 
       //retrieve custom information
       $form->_values = [];
-      CRM_Event_Form_Registration::initEventFee($form, $event['id'], FALSE);
-      CRM_Event_Form_Registration_Register::buildAmount($form, TRUE, $form->_discountId);
+      CRM_Event_Form_Registration::initEventFee($form, $event['id'], FALSE, $form->getDiscountID());
```

Comment:
@eileenmcnaughton this function doesn't seem to have a 4th function param

Reply:
@seamuslee001 https://github.com/civicrm/civicrm-core/pull/27049/files#diff-dcb99c146eb90c35dfb3a56e963fe84c57faca51fbaf9f611ca0a7922c8f509cR579